website/source/docs/upgrade-specific.html.markdown

```diff
@@ -49,7 +49,7 @@ When upgrading to Consul 0.8.0 from a version lower than 0.7.0, users will need
 set the [`-raft-protocol`](/docs/agent/options.html#_raft_protocol) option to 1 in
 order to maintain backwards compatibility with the old servers during the upgrade.
 After the servers have been migrated to version 0.8.0, `-raft-protocol` can be moved
-up to 2 and the servers restarted to match the default.
+up to 3 and the servers restarted to match the default.
```

Comment:
This is a special case for people running pre-0.7 Consul versions, so 2 was right here.